staging/src/k8s.io/controller-manager/go.mod

```diff
@@ -20,6 +20,7 @@ require (
 require (
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/antlr/antlr4/runtime/Go/antlr v1.4.10 // indirect
+	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a // indirect
```

Comment:
I'm surprised these differences showed up... do we know what dependencies this PR added? I didn't see any obvious new imports

Reply:
` $go mod why`
```
# github.com/asaskevich/govalidator
k8s.io/controller-manager/app
k8s.io/apiserver/pkg/server
k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy
k8s.io/apiserver/pkg/cel/common
k8s.io/kube-openapi/pkg/validation/strfmt
github.com/asaskevich/govalidator
```
Needs a refactor to remove `k8s.io/controller-manager/app` -> `k8s.io/apiserver/pkg/server`